actionmailbox/test/unit/test_helper_test.rb

```diff
@@ -0,0 +1,40 @@
+# frozen_string_literal: true
+
+require_relative "../test_helper"
+
+module ActionMailbox
+  class TestHelperTest < ActiveSupport::TestCase
+    test "multi-part mail can be built in tests using a block" do
+      inbound_email = create_inbound_email_from_mail do
+        to "test@example.com"
+        from "hello@example.com"
+
+        text_part do
+          body "Hello, world"
+        end
+
+        html_part do
+          body "<h1>Hello, world</h1>"
+        end
+      end
+
+      mail = inbound_email.mail
+
+      assert_equal mail.parts.length, 2
```

Comment:
It should other way around like `assert_equal expected, actual`. In this case, it should be following:

```
assert_equal 2, mail.parts.length
```
Similar for below assertions.